test/Infrastructure.IntegrationTest/Auth/Repositories/AuthRequestRepositoryTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Bit.Core.Auth.Entities;
+using Bit.Core.Auth.Enums;
+using Bit.Core.Entities;
+using Bit.Core.Repositories;
+using Xunit;
+
+namespace Bit.Infrastructure.IntegrationTest.Auth.Repositories;
+
+public class AuthRequestRepositoryTests
+{
+    private readonly static TimeSpan _userExpiration = TimeSpan.FromMinutes(15);
+    private readonly static TimeSpan _adminExpiration = TimeSpan.FromDays(6);
+    private readonly static TimeSpan _adminApprovalExpiration = TimeSpan.FromHours(12);
+
+    [DatabaseTheory, DatabaseData]
+    public async Task DeleteExpiredAsync_Works(
```

Comment:
Minor:  should this test suite be broken up into multiple, smaller tests per best practices?  Also, in the current design, if one assertion fails, then does the entire test method stop executing (which could hide other failing tests)?

Reply:
I don't think so, the purpose of this test is to both make sure it's deleting items I expect it to test as well as doesn't delete things I expect it to keep around. To do that well I would have to assert the "don't deletes" for each smaller test still but have the "should deletes" all be separate. 

Also since these are integration tests being ran against real databases that are likely to have multiple different `AuthRequest`s that need to be deleted for the various reasons I think this it's worth the larger test to more accurately test the system.

I do have the number of deleted check go last so that if you didn't delete as many as you should have you get a failure on the line for the item you either should have deleted and didn't or an item you shouldn't have deleted and did. The test failure will give you the line number and from that you can get a quick start on figuring out what you did wrong. Plus, if you have only broken one type of database but not the others it will run through all the other types still so you will know the type that you specifically broke.

Reply:
That's reasonable!